pod/dressing/templates/my_dressings.html

```diff
@@ -0,0 +1,136 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load static %}
+{% load thumbnail %}
+{% block page_title %}{% trans 'My dressings' %}{% endblock page_title %}
+{% block page_extra_head %}
+
+{% endblock page_extra_head %}
+
+{% block breadcrumbs %}
+{{block.super}}
+
+<li class="breadcrumb-item active" aria-current="page">        
+    {% trans 'My dressings' %}
+</li>
+
+{% endblock %}
+
+{% block page_content %}
+<table class="table table-sm table-striped table-bordered text-center" summary="{% trans 'List of dressings I can use or manage.' %}">
+    <caption class="caption-top">{% trans 'Available video dressings' %}</caption>
+    <thead>
+        <tr>
+            <th scope="col">{% trans 'Title' %}</th>
+            <th scope="col">{% trans 'Owners' %}</th>
+            <th scope="col">{% trans 'Users' %}</th>
+            <th scope="col">{% trans 'Allow to Groups' %}</th>
+            <th scope="col">{% trans 'Watermark' %}</th>
+            <th scope="col">{% trans 'Position' %}</th>
+            <th scope="col">{% trans 'Opacity' %}</th>
+            <th scope="col">{% trans 'Opening credits' %}</th>
+            <th scope="col">{% trans 'Ending credits' %}</th>
+            <th scope="col">{% trans 'Actions' %}</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for dressing in dressings %}
+            <tr>
+                <td>{{ dressing.title }}</td>
+                <td>
+                    {% for owner in dressing.owners.all %}
+
+                    {% if owner.first_name %}
+                        {{ owner.first_name }} {{ owner.last_name }}
+                    {% else %}
+                        {{ owner.username }}
+                    {% endif %}
+                    {% if not forloop.last %}, {% endif %}
+                    {% empty %}
+                        {% trans 'No owners' %}
+                    {% endfor %}
+                </td>
+                <td>
+                    {% for user in dressing.users.all %}
+
+                    {% if user.first_name %}
+                        {{ user.first_name }} {{ user.last_name }}
+                    {% else %}
+                        {{ user.username }}
+                    {% endif %}
+                    {% if not forloop.last %}, {% endif %}
+                    {% empty %}
+                        {% trans 'No users' %}
+                    {% endfor %}
+                </td>
+                <td>
+                    {% for group in dressing.allow_to_groups.all %}
+                        {{ group }}{% if not forloop.last %}, {% endif %}
+                    {% empty %}
+                        {% trans 'No groups' %}
+                    {% endfor %}
+                </td>
+                <td>
+                    {% if dressing.watermark %}
+                        <img id="watermark" src="{{ dressing.watermark.file.url }}" alt="{% trans 'Watermark'%}" width="50px">
```

Comment:
attention : un ID html doit toujours être unique sur une page, or ici on a `id="watermark"`à l'intérieur d'une boucle `{% for dressing in dressings %}`
Je propose de remplacver cet id par une classe `class="pod-watermark"` par exemple.

Reply:
Ah oui merci, je l'ai enlevé car de toute façon je ne l'utilise plus sans CSS.